### PR TITLE
Feat/hb/js assignment jul19

### PR DESCRIPTION
**Pull request by HarishBoke**
Added new file for JS practice

### CODE REVIEW

**Review by akkias — APPROVED** [review]
Reviewed!
Looks good to me.

File: javascript-assignment/odd-even.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function isOdd(e) {
+    return false
+}
+isOdd(8)
```
Comment (akkias) [inline javascript-assignment/odd-even.js]: @HarishBoke Can you please add an extra line to the end of the file?

**Review by akkias — COMMENTED** [review]
@HarishBoke Can you please add an extra line to the end of the file?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
 # lions-sessions
 Sessions demo and exercises files
+
+* Add description *
+-
```
Comment (HarishBoke) [inline README.md:5]: Add intentation

File: javascript-assignment/odd-even.js (hunk 1 above)
Comment (HarishBoke) [reply javascript-assignment/odd-even.js]: Sure, I will update

**Pull request merged**